tasks/src/main/scala/dagr/tasks/fgbio/CallMolecularConsensusReads.scala

```diff
@@ -36,12 +36,13 @@ import scala.collection.mutable.ListBuffer
  */
 class CallMolecularConsensusReads(val in: PathToBam,
                                   val out: PathToBam,
-                                  val rejects: PathToBam,
+                                  val rejects: Option[PathToBam] = None,
```

Comment:
I think I changed the tool in fgbio too.  I was getting tired of writing enormous rejects.bam files that I never looked at ;)